Signal/src/view controllers/MessagesViewController.m

```diff
@@ -158,12 +158,13 @@ - (void)hideInputIfNeeded {
         return;
     }
 
-    if ([_thread isKindOfClass:[TSGroupThread class]] &&
-        ![((TSGroupThread *)_thread).groupModel.groupMemberIds containsObject:[TSAccountManager localNumber]]) {
-
-        [self inputToolbar].hidden = YES; // user has requested they leave the group. further sends disallowed
-        [self.inputToolbar endEditing:TRUE];
-        self.navigationItem.rightBarButtonItem = nil; // further group action disallowed
+    if ([_thread isKindOfClass:[TSGroupThread class]]) {
+        TSGroupThread *groupThread = (TSGroupThread *)self.thread;
```

Comment:
this is just a readability change. Bug fix is in SSK. https://github.com/WhisperSystems/SignalServiceKit/pull/31